src/command_modules/azure-cli-subscription/azure/cli/command_modules/subscription/custom.py

```diff
@@ -0,0 +1,15 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.mgmt.subscription.models import SubscriptionDefinition
+
+
+def cli_subscription_create_subscription_definition(client, name, offer_type, subscription_display_name=None):
+    """Create a subscription definition."""
```

Comment:
Don't use custom command doc strings to populate help.  Instead, add a command entry to you help.py file with this as the short-summary. The reason is because the live docs edit functionality will link directly to that, and will never find it here.